packages/manager/modules/overthebox/src/overthebox/overTheBox.module.js

```diff
@@ -0,0 +1,62 @@
+import angular from 'angular';
+
+import '@ovh-ux/manager-core';
+import '@ovh-ux/manager-telecom-styles';
+import 'angularjs-scroll-glue';
+
+import '@ovh-ux/ng-ovh-telecom-universe-components';
+import '@ovh-ux/ng-ui-router-title';
+import '@ovh-ux/ng-tail-logs';
+import 'ovh-angular-responsive-tabs';
+import ngOvhUiConfirmModal from '@ovh-ux/ng-ovh-ui-confirm-modal';
+
+import 'ovh-ui-kit/dist/oui.css';
+import 'ovh-ui-kit/dist/oui-olt.css';
+import 'ovh-ui-kit-bs/dist/ovh-ui-kit-bs.min.css';
+import 'ovh-ui-kit-bs/dist/oui-bs3-olt.css';
+import 'ovh-manager-webfont/dist/css/ovh-font.css';
```

Comment:
I'm wondering if those should instead be declared as peer Dependency and be imported once in the app to avoid importing multiple time the same css dist